ssl/quic/quic_txp.c

```diff
@@ -301,6 +301,21 @@ static int tx_helper_commit(struct tx_helper *h)
         return 0;
     }
 
+    if (h->txp->args.msg_callback != NULL && l > 0) {
+        unsigned char ftype = *h->txn.data;
```

Comment:
Hmm... it's true that all the frame types that we generate in our implementation happen to have single-byte encodings for the type field. But I am uncomfortable encoding that as an assumption here. Either re-decode using the wire functions, or stash the pre-encode frame type as a `uint64_t` somewhere.

Reply:
Fixed.